devtools/db_import/cli.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+## Copyright 2023 The OpenXLA Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import argparse
+import pathlib
+import sys
+import yaml
+
+SCRIPT_DIR = pathlib.Path(__file__).resolve().parent
+sys.path.append(str(SCRIPT_DIR / "db_import"))
```

Comment:
nit: Maybe insert at the head to make sure it overrides modules with the same name?
```suggestion
sys.path.insert(0, str(SCRIPT_DIR / "db_import"))
```